documentation/documentation.gradle

```diff
@@ -180,6 +180,9 @@ task renderTopicalGuides(type: AsciidoctorTask, group: 'Documentation') {
 task renderGettingStartedGuides(type: AsciidoctorTask, group: 'Documentation') {
     description = 'Renders the Getting Started Guides (quick starts) in HTML format using Asciidoctor.'
     sourceDir = file( 'src/main/asciidoc/quickstart/guides' )
+    sources {
+        include 'index.adoc'
+    }
```

Comment:
Not sure it's a good idea to do that in a micro, but whatever. Let's at least make sure to add redirections to the 5.3 docs, from the individual files to the main one? We may have links to those documents in the wild, now that they've been published.

Reply:
I don't think we can have redirections on docs.jboss.org.

I still think it's better to remove them, considering they were well hidden and not very readable.